src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -67,6 +67,9 @@ public ReflectionMethodBodyScanner (LinkContext context, MarkStep parent)
 
 		public void ScanAndProcessReturnValue (MethodBody methodBody)
 		{
+			if (methodBody.Method.Name == "GetCSharpThunk")
+				Debug.WriteLine ("");
```

Comment:
I guess this wasn't left here intentionally?

Reply:
Removed